### ISSUE TITLE
Youtube ads don't have skip button, even with ad blocking disabled

### ISSUE THREAD

**Issue by iSasFTW** [brave/browser-laptop#8447]
- Did you search for similar issues before submitting this one?
Yes
- Describe the issue you encountered:

- Platform (Win7, 8, 10? macOS? Linux distro?):
Windows 10
- Brave Version (revision SHA):
Version 0.14.1
Rev 3de60d5
- Steps to reproduce:
    1. Open a Monetized Youtube video
    2. Make sure an ad starts playing
    3. Watch
    
- Actual result:
Ad plays entirely, regardless of length, with no option to skip. Links to advertiser's site only appear occasionally, not always. No yellow ad markers are visible on the video timeline, either.

- Expected result:
Ad starts playing, links to advertiser's site appears, skip ad button appears (can be clicked after 5 sec)

- Will the steps above reproduce in a fresh profile? If not what other info can be added?
I don't know what this means.

- Is this an issue in the currently released version?
Yes.

- Can this issue be consistently reproduced?
Yes.

- Screenshot if needed:
[Link to screenshot](https://i.supload.com/By2uqxcCl.png)

**Comment by Jacalz** [comment]
I was aple to repro with refreshing the page like 100 times juat to get nearly the same ad 👍 
In brave:
![image](https://cloud.githubusercontent.com/assets/25466657/25312664/881f023a-281f-11e7-85ba-2ba3ece96c97.png)
I Firefox:
![image](https://cloud.githubusercontent.com/assets/25466657/25312670/989c15ee-281f-11e7-8c54-6f1ccf89a4ea.png)
I chose the same video you used in the community 😉

**Comment by iSasFTW** [comment]
@Jacalz Yep, that's the exact issue. I haven't posted on the community though, in fact, I didn't know there was a forum for Brave before I saw it in the settings a few minutes ago :P 
That must have been someone else, then.

**Comment by Jacalz** [comment]
Oh there was someone from the community that posted it just yesterday or something: 😂
https://community.brave.com/t/youtube-skipping-ad-button-disappeared-brave-window-overlaps-other-screen/2329/3

**Comment by luixxiul** [comment]
@lukemulks would you mind looking at this issue?

**Comment by MrPatko0770** [comment]
Hey. Sorry for reviving an older issue, but I can confirm that this is still a thing in the latest version of Brave, on Windows 10, in exactly the same form as OP's description of the issue.

If I "Allow Ads and Tracking on YouTube", ads play but none of them can be skipped (the button simply doesn't show up, and neither does the notification saying that the video will play after the ad) and the yellow things that denote ads don't show up on the timeline either, and if I disable Shields altogether on YouTube, the same thing happens.
Compared with MS Edge, when I got the same ad, I was able to skip it, despite the fact that I had AdBlock Plus installed (just disabled on YouTube).

On a possibly unrelated note, YouTube also seems to be changing my language settings on its own as well. I live in Slovakia, so my default language is Slovak, but I'm using a VPN with a UK-based exit and I have the language manually switched to U. S. English (on Windows as well) but YouTube just randomly reverts it back to Slovak every few times I open it. Only happens in Brave though.

**Comment by luixxiul** [comment]
> I'm using a VPN with a UK-based exit and I have the language manually switched to U. S. English (on Windows as well) but YouTube just randomly reverts it back to Slovak every few times I open it. Only happens in Brave though.

I'm opening another issue for that problem.

**Comment by LaurenWags** [comment]
+1s from community: https://community.brave.com/t/youtube-ads-skip-button/22206

**Comment by GeetaSarvadnya** [comment]
+1 from community: https://community.brave.com/t/youtube-ads-unskippable/8921

**Comment by GeetaSarvadnya** [comment]
+1 from community: https://community.brave.com/t/youtube-ads-dont-have-a-skip-ad-button/27470

**Comment by sabbour** [comment]
That's still an issue till today. Any luck in the investigation?
